applications/asset_tracker/lwm2m_carrier_overlay.conf

```diff
@@ -0,0 +1,38 @@
+# To build asset tracker with LwM2M carrier lib support
+# add the following to your west build command:
+#       -DOVERLAY_CONFIG=lwm2m_carrier_overlay.conf
```

Comment:
alternatively, delete the build instructions here and just reference `applications/asset_tracker/README.rst` since that has the information?
preference?